ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/SeverityCountLabels.tsx

```diff
@@ -99,7 +128,11 @@ function SeverityCountLabels({
                                 : fadedTextColor,
                         }}
                     >
-                        {low}
+                        {!low && low !== 0 ? (
+                            <EyeSlashIcon className="pf-u-my-xs pf-u-ml-xs" />
```

Comment:
I see a combination of the EyeSlashIcon and a plain ellipsis in the mocks, and in the sketch file at the top of the epic channel it looks like most of the mocks use an ellipsis. I would check in with UX to make sure this icon is what was settled on.